motr/client.h

```diff
@@ -766,7 +766,7 @@ struct m0_client_layout {
 struct m0_idx_attr {
 	/** DIX pool layout type. */
```

Comment:
If we don't change it to "enum dix_layout_type", we can add it to comments.
```
/** DIX pool layout type. Please refer to enum dix_layout_type. */
```

Reply:
Done.